CONTRIBUTING.md

```diff
@@ -139,7 +139,16 @@ A guide on commonly used labels added to issues and pull requests:
 
 ## Testing Your Code
 
-- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser .
+- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser.
+
+This project contains Jest tests. These tests can be run using `yarn test`. When new features are added, a Jest test should also be added. The following best practices should be observed when writing Jest tests:
+
+- `screen` should be used for querying.
```

Comment:
When the reader gets to this point, it would be helpful to know that React Testing Library is what is being referred to here.

Reply:
Also if theres an example of a "good" test, it might be helpful to link to it here.

Reply:
In latest commit I clarified that the rules/guidelines are related to react testing library.

As far as including examples of good tests I was thinking of potentially going through a smaller component and re-writing the tests for it so they all conform with the best practices and then we could reference that whole test file as an example? For now the Common mistakes with React Testing Library article includes examples. But I agree it would be nice to have grommet examples.